CHANGELOG.rst

```diff
@@ -1,5 +1,61 @@
 Changelog
 =========
+1.13 (2020-12-22)
+-----------------
+MLflow 1.13 includes several major features and improvements:
+
+Features:
+
+New fluent APIs for logging in-memory objects as artifacts:
+
+- Add ``mlflow.log_text`` which logs text as an artifact (#3678, @harupy)
+- Add ``mlflow.log_dict`` which logs a dictionary as an artifact (#3685, @harupy)
+- Add ``mlflow.log_figure`` which logs a figure object as an artifact (#3707, @harupy)
+- Add ``mlflow.log_image`` which logs an image object as an artifact (#3728, @harupy)
+
+UI update / fixes (#3867, @smurching):
+
+- Add "Empty state" overlay for experiments
+- Add ``SectionErrorBoundary`` for better error handling
+- Use local storage to persist model registry UI settings
+- Add ``data-test-id`` / DOM classname to facilitate e2e UI testing
```

Comment:
Lines 19 and 21 here don't seem like noteworthy features. 19 is more of a bug fix and 21 is a dev infrastructure item that probably shouldn't be called out explicitly

Reply:
Removed both of them and added #3867 (which made these changes) in `small changes`.